.circleci/config.yml

```diff
@@ -18,37 +18,35 @@ checkout step for each job: &addWorkspace
 default config for js: &js_defaults
   <<: *defaults
   docker:
-    - image: circleci/node:14
+    - image: cimg/node:14.18.1
 
 default config for macOS: &macos_defaults
   <<: *defaults
   resource_class: 'medium'
   macos:
-    xcode: '13.2.1'
+    xcode: '13.1.0'
 
-config for macOS (android): &macos_defaults_android
+config for android e2e: &android_e2e
   <<: *defaults
-  resource_class: 'medium'
-  macos:
-    xcode: '11.5.0'
+  resource_class: large
+  machine:
+    image: android:2022.06.2
 
 default config for android apk builds: &android_defaults
   <<: *defaults
   docker:
-    - image: reactnativecommunity/react-native-android
-  resource_class: 'medium'
-  working_directory: ~/async_storage
+    - image: cimg/android:2021.10.2-node # node 14.18.1
```

Comment:
Shouldn't this also use `cimg/node:14.18.1` as above? Or `android:2022.06.2`?

Reply:
This image has Android SDK preinstalled, and is used to create an apk for tests. I could merge the apk creation into e2e job, since it's no longer macos.

Reply:
Is there a cost to spinning up a separate machine? I'd imagine we should stay on the same machine for as long as possible, but I don't know how CircleCI works.

Reply:
I think there's no cost for boot up, but rather the overall execution time.